src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer is started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer is stopped working");

```

Comment:
Similarly, the stop message should be more informative. For example, 'Bulldozer has stopped its work.'